run.sh

```
@@ -337,6 +337,39 @@ elnbuildsync \
+# bash immediately, leaving elnbuildsync running as an orphan until the
+# runtime gives up waiting and SIGKILLs the whole cgroup -- Twisted's
+# reactor never gets a chance to shut down cleanly. Forward it (and SIGINT,
+# for interactive `podman run`/Ctrl-C) to the real daemon instead.
+#
+# exec'ing into elnbuildsync instead of backgrounding it would be simpler,
+# but would also skip the Kerberos ccache cleanup EXIT trap set up above.
+_term_received=0
+_forward_signal() {
+  _term_received=1
+  kill -TERM "${_elnbuildsync_pid}" 2>/dev/null || true
+}
```

Shellcheck (0.11.0)

[info] This function is never invoked. Check usage (or ignored if invoked indirectly).

(SC2329)